src/types.rs

```diff
@@ -185,6 +187,7 @@ pub enum BinOper {
     RShift,
     As,
     Escape,
+    Custom(DynIden),
```

Comment:
My question is why we would like to reuse DynIden here?
It was originally intended for 'identifiers'.
May be we can have a new type CustomOper, which is simply a wrapped `&'static str`

Reply:
Right, I was not aware we need to remove the `Copy` derive. So, yes, `&'static str` is a better alternative.

@rakbladsvalsen what do you think?

Reply:
Thanks for the feedback guys! This PR _should_ be good to go.